File: src/main/java/app/askresume/api/resume/dto/response/WhatGeneratedResponse.java
package app.askresume.api.resume.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;

import java.util.ArrayList;
import java.util.List;

public record WhatGeneratedResponse(
        @Schema(description = "예상 질문 리스트", required = true)
        List<PredictionResponse> predictionResponse


) {
    public WhatGeneratedResponse() {
        this(new ArrayList<>());
    }
}

File: src/main/java/app/askresume/api/resume/dto/response/WhatGeneratedResponse.kt
package app.askresume.api.resume.dto.response

import io.swagger.v3.oas.annotations.media.Schema

data class WhatGeneratedResponse(

    @field:Schema(
        description = "예상 질문 리스트",
        required = true
    )
    val predictionResponse: List<PredictionResponse>

) {
    constructor() : this(ArrayList())
}

